elm-sdks/morphir-unit-test/src/Morphir/UnitTest/Expect.elm

```diff
@@ -0,0 +1,120 @@
+module Morphir.UnitTest.Expect exposing(..)
+import Morphir.SDK.Dict exposing(Dict, toList, get)
+import Morphir.SDK.List exposing(List)
+import Morphir.SDK.Set exposing(Set)
+
+type Expectation = Expectation ExpectationResult
```

Comment:
Why is "ExpectationResult" wrapped in an "Expectation"?
I'm probably missing something, but it seems like you could just define
`type Expectation = Pass | Fail String`

Reply:
That fell out of me trying to keep a level of abstraction around the `Expectation` type. I'd honestly rather keep it in place, some of the back-end code is built around things being referred to that way.